### ISSUE TITLE
Customize the color of mute reason text &/or background

### ISSUE THREAD

**Issue by ghost** [ScratchAddons/ScratchAddons#6972]
EDIT: Made the issue on scratch-www, I presume I'll wait now.
EDIT2: Been a week since the issue was made - It's basically gone unnoticed there.

### Why this would be helpful

~Currently with #6873 it has turned yellow, however the text is white.~ (Its replicating how "You can't post an empty comment!" works my bad)
The two colors mix in a lot and it's kind of hard to read as a mix (I probably should've suggested possibly since I even saw that PR)

This also occurs with login-screen errors.

### My suggestion

~Allow the ability to customize it in `dark-www` (mute background/mute text color)~ (striked in edit)
(added in edit) Use the notifications setting to recolor the background of the mute.

### Possible alternatives

1. Decrease the contrast
2. Change the colors of the background
3. Change the colors of both

### Additional context

![example](https://github.com/ScratchAddons/ScratchAddons/assets/150537842/c693f25c-4a71-4218-bbbc-995f3151f305)

**Comment by mybearworld** [comment]
Maybe it could just be the color used for messages?

**Comment by WorldLanguages** [comment]
Yeah these colors are hard to see. Not sure why Scratch chose them.

**Comment by mybearworld** [comment]
Try posting an empty comment on a scratch-www page. You'll see that same shade of orange.

**Comment by BroJac5246** [comment]
> Allow the ability to customize it in dark-www (mute background/mute text color)
I prefer mybearworld's suggestion— users don't see this text enough to justify, in my opinion, adding a whole new setting for it.

> Yeah these colors are hard to see. Not sure why Scratch chose them.
Since this is a Scratch issue, would it be better to create a suggestion or PR for Scratch directly (note for if we do this: this issue is also present for the sign-in field)

**Comment by mybearworld** [comment]
> > Try posting an empty comment on a scratch-www page. You'll see that same shade of orange.
> 
> As note, this also happens with the login screen if you enter a wrong password/username, which I just realized. Gonna try making an issue today

I think this is basically the same issue.

**Comment by WorldLanguages** [comment]
@DNin01 I think this issue might be interesting to you. Sorry for pinging you if you've already read this one.

**Comment by DNin01** [comment]
Related: #6177
